lib/funneler/version.rb

```diff
@@ -0,0 +1,3 @@
+module Funneler
+  VERSION = "1.0"
```

Comment:
@fgrehm - This gem has been incubated within Doximity app for about 3mo now and is fairly stable.  You can see its extraction here: https://github.com/doximity/doximity/issues/16845